UPGRADE-RUNBOOK.md

```
@@ -130,39 +144,85 @@ cast call <PROXY_ADDRESS> "balanceOf(address,uint256)(uint256)" <HOLDER_ADDRESS>
+
+forge script script/FabricaTokenDeployImpl.s.sol \
+  --sig "run(address)" "$PROXY_ADDRESS" \
+  --rpc-url "$RPC_NETWORK_NAME" \
   --broadcast \
+  --verifier etherscan \
   --verify \
   --account "$TESTNET_DEPLOYER_ACCOUNT"
 ```
 
 Example for Sepolia:
 ```bash
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)